bundles/org.eclipse.osgi.service.http.servlet.contract/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,17 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: JavaServlet OSGi Contract Provider for Equinox
+Bundle-SymbolicName: org.eclipse.osgi.service.http.servlet.contract;singleton:=true
+Bundle-Version: 1.0.0.qualifier
+Fragment-Host: 
+ jakarta.servlet-api;bundle-version="[4.0.0,5.0.0)",
+ javax.servlet-api;bundle-version="[3.0.0,5.0.0)",
+ org.eclipse.jetty.servlet-api;bundle-version="[4.0.0,5.0.0)"
+Automatic-Module-Name: org.eclipse.osgi.service.http.servlet.contract
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Provide-Capability: osgi.contract;osgi.contract=JavaServlet
+ ;version:List<Version>="2.6,3.0,3.1,4.0"
```

Comment:
According to https://docs.osgi.org/reference/portable-java-contracts.html the `osgi.contract=JavaServlet` is specified with version `2.5`, `3.0`, `3.1` and `4.0`. 
But [`org.apache.felix:org.apache.felix.http.servlet-api:1.2.0`](https://mvnrepository.com/artifact/org.apache.felix/org.apache.felix.http.servlet-api/1.2.0) defines 2.6 instead of 2.5.

Reply:
Changed the list to have the official version. Since o.o.service.http.whiteboard only uses 3.1 all the others doesn't matter for our use-case anyway.